server/src/test/java/org/elasticsearch/index/engine/LuceneChangesSnapshotTests.java

```diff
@@ -178,9 +187,10 @@ public void testUpdateAndReadChangesConcurrently() throws Exception {
         }
         boolean onPrimary = randomBoolean();
         List<Engine.Operation> operations = new ArrayList<>();
-        int numOps = scaledRandomIntBetween(1, 1000);
+        int numOps = frequently() ? scaledRandomIntBetween(1, 1000) : scaledRandomIntBetween(5000, 20_000);
+        boolean updateFrequently = rarely();
```

Comment:
I am not sure I follow why this should be `rarely()` as opposed to just `randomBoolean()`?